cms/templates/category.jinja2

```diff
@@ -0,0 +1,12 @@
+{% extends "base.jinja2" %}
+{% import 'atoms.html' as atoms %}
+{% import 'molecules.html' as molecules %}
+{% import 'organisms.html' as organisms %}
+
+{% block content %}
+
+{{molecules.image_and_heading(view, category)}}
+
+{{organisms.list_of_links_FFL(pages, category)}}
```

Comment:
Not sure how you guys did stuff on _fufu_ but it doesn't feel right to have app-specific macros here. It feels like this should just be `{{organisms.list_of_links(pages, category)}}` or `{{organisms.list_of_links_menu(pages, category)}}`